data/Archlinux.yaml

```diff
@@ -4,11 +4,12 @@ prometheus::server::manage_user: false
 prometheus::server::manage_group: false
 prometheus::server::package_ensure: 'installed'
 prometheus::server::manage_config_dir: false
-prometheus::server::manage_init_file: false
+prometheus::server::manage_init_file: true
 prometheus::configname: 'prometheus.yml'
-prometheus::version: '2.46.0'
-prometheus::install_method: 'package'
-prometheus::bin_dir: '/usr/bin'
+prometheus::server::version: '2.51.2'
+prometheus::server::install_method: 'package'
+prometheus::server::init_style: 'systemd'
```

Comment:
mhm do we need to explicitly set this?

Reply:
Need to re-check. There is still an issue here. unit file uses $PROMETHEUS_ARGS but /etc/conf.d/prometheus sets $ARGS

Reply:
Ah right, the module writes /etc/systemd/system/prometheus.service correctly. So the package unit-file is not used.
Also works without seeting the init_style explicilty. I will remove it